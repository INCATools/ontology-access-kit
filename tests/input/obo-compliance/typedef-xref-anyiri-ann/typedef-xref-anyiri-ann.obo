format-version: 1.4
ontology: typedef-xref-anyiri-ann

[Typedef]
id: R:1
xref: http://example.org/R/1
is_metadata_tag: true

