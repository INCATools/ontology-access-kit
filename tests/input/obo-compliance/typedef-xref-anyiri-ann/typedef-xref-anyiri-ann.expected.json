{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/typedef-xref-anyiri-ann.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://purl.obolibrary.org/obo/R_1",
      "type" : "PROPERTY",
      "meta" : {
        "xrefs" : [ {
          "val" : "http://example.org/R/1"
        } ],
        "basicPropertyValues" : [ {
          "pred" : "http://www.geneontology.org/formats/oboInOwl#is_metadata_tag",
          "val" : "true"
        } ]
      }
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasDbXref",
      "lbl" : "database_cross_reference",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    } ]
  } ]
}